src/native-common/utils/EventHelpers.ts

```diff
@@ -189,6 +183,19 @@ export class EventHelpers {
         return mouseEvent;
     }
 
+    toMouseButton(e: Types.TouchEvent): number {
+        const nativeEvent = e as any;
+        if (nativeEvent.button !== undefined) {
+            return nativeEvent.button;
+        } else if (nativeEvent.isRightButton) {
```

Comment:
Are you sure "isRightButton" shouldn't be capitalized? It was in the old code. Same with IsMiddleButton. Was the old code incorrect?